packages/OmegaPrint-Core.package/OPPrinter.class/instance/value..st

```diff
@@ -0,0 +1,25 @@
+value
```

Comment:
Ich waere bei dieser Methode fuer eine extension Methode, um den Vorteil der Memoization nicht zu verlieren

Reply:
Noch viel besser als eine extension Methode koennte auch sein, in dieser Methode einfach `super value: inputNode` zu callen und dann anschliessend einfach unseren Kommentar davorzuhaengen. Zumindest bei den super calls haetten wir dann memoization.

Reply:
gute Idee